### ISSUE TITLE
Missing php-index for malkusch/bav dependency

### ISSUE THREAD

**Issue by brunoluiz** [Respect/Validation#885]
I am having issues to install the project dependencies due to missing php-index package:

```
➜  Validation git:(master) composer update --prefer-dist --prefer-lowest --prefer-stable
Loading composer repositories with package information
Updating dependencies (including require-dev)
  - Removing egulias/email-validator (2.1.2)
  - Installing egulias/email-validator (2.0.0)
    Downloading: 100%

  - Installing malkusch/php-index (0.1.0)
    Downloading: Failed
    Failed to download malkusch/php-index from dist: The "https://api.github.com/repos/malkusch/php-index/zipball/7961603d647e5da9839e4eae127e810748a6591f" file could not be downloaded (HTTP/1.1 404 Not Found)
    Now trying to download from source
  - Installing malkusch/php-index (0.1.0)
    Cloning 7961603d647e5da9839e4eae127e810748a6591f


  [RuntimeException]
  Failed to clone https://github.com/malkusch/php-index.git via https, s
  sh protocols, aborting.
  - https://github.com/malkusch/php-index.git
    Cloning into '/Users/brunoluiz/git/Validation/vendor/malkusch/php-in
  dex'...
    remote: Invalid username or password.
    fatal: Authentication failed for 'https://github.com/malkusch/php-in
  dex.git/'
  - git@github.com:malkusch/php-index.git
    Cloning into '/Users/brunoluiz/git/Validation/vendor/malkusch/php-in
  dex'...
    ERROR: Repository not found.
    fatal: Could not read from remote repository.

    Please make sure you have the correct access rights
    and the repository exists.
```

The issue is that `malkusch` seems to have deleted the `bav` and `php-index` repos: https://github.com/malkusch/php-index | https://github.com/malkusch/bav.

**Comment by henriquemoody** [comment]
I suggest you to remove @malkusch's packages from your dependencies if you're not using it. Validation doesn't need it to work, it's just a suggested package.

**Comment by malkusch** [comment]
Oh, Sorry guys. I didn't expect breaking packagist by removing those repos.

**Comment by henriquemoody** [comment]
Fixed on [1.0.20](https://github.com/Respect/Validation/releases/tag/1.0.20) and [1.1.15](https://github.com/Respect/Validation/releases/tag/1.1.15).

Thanks for reporting! :panda_face:

It's all fine now, @malkusch. Now I just wonder why did you remove the packages 👀
